### PR TITLE
#54 - Added API version header to version check response

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #54 
Added "Docker-Distribution-API-Version" header to version check response. It is required for client to know that registry works by 2.0 protocol

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#71](https://github.com/artipie/docker-adapter/issues/71) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/2b24595b-7b82-4be3-9f32-d110ddb8be87); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/docker/http/VersionCheck.java [hunk 1]
```diff
@@ -40,8 +43,15 @@
...
-        final String line, final Iterable<Map.Entry<String, String>> headers,
-        final Publisher<ByteBuffer> body) {
-        return new RsWithStatus(RsStatus.OK);
+        final String line,
+        final Iterable<Map.Entry<String, String>> headers,
+        final Publisher<ByteBuffer> body
+    ) {
+        return new RsWithHeaders(
+            new RsWithStatus(RsStatus.OK),
+            Collections.singleton(
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/docker/http/VersionCheck.java:52]: @olegmoz there is a constructor specifically for providing a single header entry. Check this: https://github.com/artipie/http/blob/master/src/main/java/com/artipie/http/rs/RsWithHeaders.java#L74

Comment (olegmoz) [reply src/main/java/com/artipie/docker/http/VersionCheck.java:52]: @carlosmiranda thank you! Fixed

**Comment by 0crat** [comment]
Job was finished in 44 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 12@9452952fc9d1, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-boosts.xsl says: The boost for job "gh:artipie/docker-adapter#71" exists in boosts.xml, but the job is absent in the WBS among 37 jobs.
```

[0.53](https://github.com/zerocracy/farm): CID: [a161caeb-bdd9-45d2-8de7-969102b5de2b](https://www.0crat.com/footprint/CT2E6TK9B/a161caeb-bdd9-45d2-8de7-969102b5de2b), Type: "Remove job from WBS"

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will find at least 3 issues during next CR, thanks!

**Comment by carlosmiranda** [comment]
@sereshqua confirmed

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable  ([here](https://github.com/artipie/docker-adapter/issues/71#issuecomment-611876450))

@sereshqua The project doesn't have enough funds, can't make a payment

**Pull request merged**